test/views/ExclusionOverviewViewSpec.scala

```diff
@@ -21,22 +21,45 @@ import play.api.i18n.MessagesApi
 import play.twirl.api.Html
 import utils.FormMappings
 import views.helper.PBIKViewSpec
+import org.jsoup.Jsoup
 import views.html.exclusion.ExclusionOverview
 
 class ExclusionOverviewViewSpec extends PBIKViewSpec {
 
   val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
-  val formMappings = app.injector.instanceOf[FormMappings]
-  val exclusionOverviewView = app.injector.instanceOf[ExclusionOverview]
+  val formMappings: FormMappings = app.injector.instanceOf[FormMappings]
+  val exclusionOverviewView: ExclusionOverview = app.injector.instanceOf[ExclusionOverview]
 
-  def taxYearRange = TaxYearRange(2018, 2019, 2020)
+  def taxYearRange: TaxYearRange = TaxYearRange(2018, 2019, 2020)
 
-  override def view: Html = exclusionOverviewView(taxYearRange, "cyp1", "30", List(), EmpRef("", ""))
+  private val iabdType = "31"
```

Comment:
should this value have changed to `31`?

Reply:
not required.. this is just a test so we can have any value